datastore.go

```diff
@@ -618,7 +618,7 @@ func (t *txn) query(q dsq.Query) (dsq.Results, error) {
 
 			matches := true
 			check := func(value []byte) error {
-				e := dsq.Entry{Key: string(item.Key()), Value: value}
+				e := dsq.Entry{Key: string(item.Key()), Value: value, Size:len(value)}
```

Comment:
`value` is `nil` for `KeysOnly`. We'll want to do this as a separate step inside

https://github.com/ipfs/go-ds-badger/pull/76/files#diff-c587b3d69dfe75cd43314ecab24f0af5R633-R636

Reply:
Hm. Actually, we can just call `ValueSize` on the item directly. Also, this doesn't really matter because this is the "skip" part.

Reply:
(fixed)